gradle/libs.versions.toml

```diff
@@ -1,6 +1,6 @@
 [versions]
 gradle = "8.2.1"
-kotlin = "1.9.22"
+kotlin = "2.0.0"
 dokka = "1.9.20"
 android = "8.1.1"
 jvmTarget = "1.8"
```

Comment:
Can you use this value instead of `JvmTarget.JVM_1_8`?
This way, we can update all the places at the same time when needed.

Reply:
Hi 
Actually the problem is they have moved from String definition to enum definition for JVM Target. We have to use enum value. I don't see a way to define enum type in libs.version.toml

Reply:
I see, thanks.

Then, can you update other places using this value(like here: https://github.com/yshrsmz/BuildKonfig/blob/master/buildkonfig-compiler/build.gradle.kts#L44) and remove this `jvmTarget` ?

Reply:
sure

Reply:
Hi, 
I updated compiler options in the buildKonfig-compiler module.